packages/react-bootstrap-table2/test/props-resolver/index.test.js

```diff
@@ -99,48 +100,292 @@ describe('TableResolver', () => {
         expect(cellEdit.cidx).toEqual(cidx);
       });
     });
+
+    describe('if cellEdit prop defined', () => {
```

Comment:
help to change describe as `resolveCellEditProps`

Reply:
Sorry I couldn't understand. Function `resolveCellEditProps` deal with prop `cellEdit` so I thought there is nothing improper.

Reply:
oh, my mistake, nothing here